src/notation/internal/notationinteraction.cpp.uncrustify

```diff
@@ -0,0 +1,3918 @@
+/*
```

Comment:
That file doesn't blong into the repo at all, and causes the GitHub CI codestyle failure